### DISCUSSION TITLE
JSON Diffs?

### DISCUSSION THREAD

**Discussion by fangmarks** [FRSOURCE/cypress-plugin-visual-regression-diff#93] [Q&A] [answered]
I was wondering if this module supports showing diffs in json, as that's my main use case.

**Comment by fangmarks** [comment, marked as answer]
Seeing as how my own testing and a look into the commands showed me that only images are supported, I'll answer this myself:

no

**Reply by FRSgit** [reply]
Hi @himbolion!

Exactly, your use case seems to be out of the scope of this package 😞 
If you need to test and compare text/json snapshots there should be some other cypress plugin available for that.

**Reply by fangmarks** [reply]
I sadly haven't found any that fit my use-case without them needing some kind of modification done to them to so what i want them to so

**Reply by FRSgit** [reply]
I think that you should give some attention to https://github.com/cypress-io/cypress/issues/4084, there are some solutions in there and also - you can always let Cypress team know that it would be an important feature for you
